### ISSUE TITLE
Allocating specific AIDS conditions 

### ISSUE THREAD

**Issue by andrew-phillips-1** [UCL/TLOmodel#1170]
Hi @tdm32 On the epidemiology call yesterday we were talking about the fact that we were not capturing the high number of deaths from cryptococcal meningitis in the GBD.  We realise that this is likely just because we don't try to split up AIDS events into the actual constituent conditions.  We were wondering about simply allocating a specific AIDS condition for each non-TB AIDS event that occurs based on data on the risk of each.  I'm not suggesting that we would model separate HSIs for each separate condition to reflect the different treatments (e.g. for Kaposi Sarcoma versus cryptococcal meningitis) but I guess that's not impossible as a future development.

**Comment by tbhallett** [comment]
Thanks @andrew-phillips-1 - could possibly be a good student project for someone at ICL or UCL.

**Comment by tdm32** [comment]
Hi @andrew-phillips-1 , yes certainly. Depending on how detailed we need to be:
- we can classify deaths by sub-cause matching GBD / other sources
- we can stratify AIDS by specific conditions, all with the same survival rate
- we could add referral to one HSI upon onset of AIDS (triggered by symptoms or during routine monitoring) which would look at the AIDS-defining condition and allocate appropriate treatment

I think it's certainly doable without too much additional modelling, I suppose we just need clear direction on how to split conditions and the impacts of specific treatments if we were to go down that route. 

Possible project for a masters student (the data scoping)?

**Comment by andrew-phillips-1** [comment]
Sounds good @tdm32 If I follow code correctly, anyone developing aids symptoms will have a time of death assigned but this is cancelled if they become virally suppressed on ART.  If there were to be specific HSIs for each AIDS condition this would open up the challenge of deciding what probability the treatment would have of cancelling the AIDS symptoms.  I think maybe best to simply start by allocating a label (of which AIDS they have) for each person with AIDS symptoms.  Or in fact it might not be necessary to do this until they die from their AIDS symptoms ?  That would be the simplest approach ?

**Comment by tdm32** [comment]
Yes I think the simplest way to start would be to distribute AIDS cases across the ADC according to any data that we can find. Then we can think about which interventions we need to include. 

I'd like to keep it simple as we'd likely to have do some re-calibration in order to preserve the overall mortality rates. Happy to do this of course, but I'd like to do it in small steps as I worry that including many different treatments for the various aids-defining illnesses could quickly become incredibly tricky to calibrate. 

Perhaps we could start with KS and cryptococcal meningitis then consider pneumonia/CMV...?

**Comment by andrew-phillips-1** [comment]
I agree with keeping it simple and I don't want you to have to re-calibrate.  So I was thinking we could just allocate the specific AIDS condition at death with no further changes at all.  So we would not know the specific AIDS condition for living people, only those that have died.

**Comment by tdm32** [comment]
Ok - this would be a pretty simple edit in that case. But did you want to capture any specific interventions other than first-line ART?

**Comment by andrew-phillips-1** [comment]
I think that would be a secondary consideration so I suggest we don't worry about doing that for now.  If in future a person wants to study effects of interventions for specific AIDS diseases they might have to make the change at that point.

**Comment by tdm32** [comment]
Ok I'll raise an issue for this - thanks Andrew
